src/linker.zig

```diff
@@ -621,7 +619,7 @@ pub const Linker = struct {
         }
 
         if (import_record.path.text.len > 0 and Resolver.isPackagePath(import_record.path.text)) {
-            if (linker.options.target.isWebLike() and Options.ExternalModules.isNodeBuiltin(import_record.path.text)) {
+            if (linker.options.target == .browser and Options.ExternalModules.isNodeBuiltin(import_record.path.text)) {
```

Comment:
intentional?

Reply:
yes, i removed some methods after realizing they had one or two total call sites, and who's method name implied comparison with a single target.